lib/lighthouse.py

```diff
@@ -0,0 +1,67 @@
+import json
+import subprocess
+from urllib.parse import urlparse
+from datadog import DataDogApiClient
+
+class Lighthouse:
+    command = ['lighthouse', '--output=json', '--quiet', '--chrome-flags="--headless"']
+
+    def run(self, url):
+        command = self.command + [url]
+        result = subprocess.run(command, stdout=subprocess.PIPE)
+
+        if result.returncode != 0:
+            raise Exception('Lighthouse failed')
+
+        json_results = json.loads(result.stdout)
+        metrics = self.parse_json_report(json_results)
+        self.send_metrics_to_datadog(url, metrics)
```

Comment:
I'd recommend these lines be in a function in `main.py`, simply as a separation of concerns measure: I think the code will be easier to work with if `lighthouse.py` only interfaces with `lighthouse` and `main.py` does the stitching-together of `lighthouse.py` and `datadog.py`.